lib/linalg/Matrix.cpp

```diff
@@ -1790,46 +1790,95 @@ const
 void
 Matrix::orthogonalize()
 {
-    for (int work = 1; work < d_num_cols; ++work) {
+    for (int work = 0; work < d_num_cols; ++work)
```

Comment:
This is the real change in this function; ensuring that the first column is normalized. The rest is changes to make the code easier to read; no functionality changes.